examples/tree-view/containers/Node.js

```diff
@@ -23,16 +24,23 @@ class Node extends Component {
     addChild(id, childId)
   }
 
+  handleRemoveChildClick(childId, e) {
+    e.preventDefault();
+
+    const { removeChild, id } = this.props
+    removeChild(id, childId)
+  }
+
   renderChild(childId) {
     return (
       <li key={childId}>
-        <ConnectedNode id={childId} />
+        <ConnectedNode id={childId} handleRemoveChildClick={this.handleRemoveChildClick.bind(this, childId)} />
```

Comment:
Binding is going to be expensive here because it's called a lot of times.
The child already knows its ID so there is no need to bind here—it can supply it as an argument by reading the props.